packages/mobile/src/screens/chat-screen/ChatMessageSeparator.tsx

```diff
@@ -34,7 +32,14 @@ export const ChatMessageSeparator = ({ content }: { content: string }) => {
   return (
     <View style={styles.unreadTagContainer}>
       <View style={styles.unreadSeparator} />
-      <Text style={styles.unreadTag}>{content}</Text>
+      <Text
```

Comment:
Is there a default color on Text if you don't specify one? I saw below that other usage included a `color` prop.

Reply:
yep it defaults to `neutral`.